microsoft/testsuites/acc/bvt.py

```diff
@@ -0,0 +1,191 @@
+from assertpy import assert_that
+
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from lisa.features import acc
+from lisa.operating_system import Debian
+from lisa.tools import Dmesg, Make, Wget, Whoami
+from lisa.util import SkippedException
+
+
+@TestSuiteMetadata(
+    area="ACC_BVT",
+    category="functional",
+    description="""
+    This Basic Validation Test (BVT) suite validates the availability of Secure Guard \
+    Extensions (SGX) on a given
+    platform.
+    """,
+)
+class ACCBasicTest(TestSuite):
+    @TestCaseMetadata(
+        description="""
+
+        This case verifies if the VM is SGX Enabled.
+
+        Steps:
+            1. Add keys and tool chain from Intel-SGX, LLVM and Microsoft repositories.
+            2. Install DCAP driver if missing.
+            3. Install required package.
+            4. Run Helloworld and Remote Attestation tests.
+
+        """,
+        priority=1,
+        requirement=simple_requirement(
+            supported_features=[acc.ACC],
+        ),
+    )
+    def verify_sgx(self, log: Logger, node: Node) -> None:
+
+        if node.os.information.version == "18.4.0":
+            os_version = "18.04"
+        elif node.os.information.version == "20.4.0":
+            os_version = "20.04"
+        else:
+            raise SkippedException(node.os)
```

Comment:
Can you please add complete message. Something like "os version: {} is not supported"